### DISCUSSION TITLE
Нужна помощь

### DISCUSSION THREAD

**Discussion by Mixail202457** [blockscout/blockscout#10245] [Support]
Здравствуйте. Не могли бы вы мне помочь? При покупке EFC по ошибке отправил адрес кошелька EFH. Как быть?

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
